catalog.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Каталог</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<ul>
+				<a href="index.html"><img src="img/logo.svg" alt="logo"></a>
+				<li><a href="catalog.html">Информация</a></li>
+				<li><a href="#">Фото и видео</a></li>
+				<li><a href="#">Карта штата</a></li>
+				<li><a href="#">Гостиницы</a></li>
+			</ul>
+		</nav>
+	</header>
+	<section>
+		<h1>Каталог</h1>
+		<form>
+			<label>Инфраструктура:</label><br>
+			<input type="checkbox" checked name="Бассейн">
+			<label>Бассейн</label><br>
+			<input type="checkbox" name="Парковка">
+			<label>Парковка</label><br>
+			<input type="checkbox" name="Wi-Fi">
+			<label>Wi-Fi</label><br>
+			<label>Тип жилья:</label><br>
+			<input type="checkbox" checked name="Гостиница">
+			<label>Гостиница</label><br>
+			<input type="checkbox" checked name="Мотель">
+			<label>Мотель</label><br>
+			<input type="checkbox" checked name="Аппартаменты">
+			<label>Аппартаменты</label><br>
+			<label>Стоимость в сутки</label>
+			<input type="range" min="0" max="3000" step="1" value="1500">
+		</form>
+	</section>
+	<section>
+		<h2>Найдено: 3</h2>
+		<p>Сортировка:</p>
+		<p>По цене</p>
+		<p>По типу</p>
+		<p>По рейтингу</p>
+		<ul>
+			<li><h3>Amara Resort & Spa</h3></li>
```

Comment:
Не просто заголовок, а ещё и ссылка (то, что это ссылка, можно понять по состоянию элемента в стайлгайде).
```suggestion
			<h3><a href="#">Amara Resort & Spa</a></h3>
```